tests/test-remove-ad-server.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(cd "$(dirname "${0}")" && pwd)"
+
+source "${SCRIPT_DIR}/common.sh"
+
+if [ ${KEEP} -eq 1 ]; then
```

Comment:
Should this be named something more specific like KEEP_AD or KEEP_AD_DEPLOYMENT?

Reply:
Well, I was thinking about this.
The current patch uses `$KEEP` in both `test-remove-ad-server.sh` and `test-remove-ad-member.sh`.
I could introduce two separate variables...

🤔

Reply:
kept is as one var for now